### PR TITLE
Handle mySNU login request

### PR DESCRIPTION
**Pull request by Jhuni0123**
Send mysnu login request when click login button.

TODO:
- backend
    - modify major info api definition ( when `major_info` is `true`)
        - ex) `"majors": [{"type": "주전공", "field" : "컴퓨터공학전공"}, ...]` 
- frontend
    - handle major list when `major_info` is `true`

### CODE REVIEW

File: backend/api/login/views.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def post(self, request, option):
         # TODO: fix error ( tree is not serializable )
         # request.session['tree'] = tree
         request.session.set_expiry(6000)
-        return JsonResponse({'success': True, 'message': str(request.session['list'])})
+        return JsonResponse({'success': True, 'message': request.session['list']})
```
Comment (sonowz) [inline backend/api/login/views.py:65]: What is the reason of deleting `str` here?

**Comment by sonowz** [comment]
Have you tested that frontend gets the response correctly? If so, I'll approve.
Then I'll update API at #28, with changing `url-encoded` request parameters to `JSON`.

File: backend/api/login/views.py (hunk 1 above)
Comment (Jhuni0123) [reply backend/api/login/views.py:65]: Oh,  I didn't think about that it is just a message. my mistake.

**Comment by Jhuni0123** [comment]
Yes, we can login by mysnu successfully.

File: backend/api/login/views.py (hunk 1 above)
Comment (sonowz) [reply backend/api/login/views.py:65]: Oh, and I just realized that sending `request.session['list']` is for debugging.
You don't need to parse it if you don't want to.

**Review by sonowz — APPROVED** [review]
Then I'll merge this.

**Pull request merged**